agent/api/task/task.go

```diff
@@ -163,6 +165,8 @@ const (
 	serviceConnectAttachmentType = "serviceconnectdetail"
 
 	ipv6LoopbackAddress = "::1"
+
+	PAUSE_LABELS_ENV_VAR = "ECS_PAUSE_LABELS"
```

Comment:
nit: Golang convention is to use camel case for naming (i.e., `pauseLabelsEnvVar`).